Tests/ExampleAnalyzers/RoslynAnalyzer10/RoslynAnalyzer10/ExampleAnalyzer2.cs

```diff
@@ -29,6 +29,9 @@ public class ExampleAnalyzer2 : DiagnosticAnalyzer
         private static readonly LocalizableString Description = "ExampleAnalyzer2 Description";
         private const string Category = "Naming";
 
+        // Referencing this forces some commonly-used libraries to be loaded
+        private static readonly LanguageVersion csVersion = Microsoft.CodeAnalysis.CSharp.LanguageVersion.CSharp1;
```

Comment:
We need to directly reference something inside a library in order to force it to load.

Previously we were importing Microsoft.CodeAnalysis.CSharp, but it wasn't being used.